dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -46,9 +46,6 @@
 		clk_hse: clk-hse {
 			#clock-cells = <0>;
 			compatible = "st,stm32-hse-clock";
-			/* H7 clock driver may not always need this value */
```

Comment:
As per former comment, I understand that the property is required by binding, even if board doesn't have HSE at all.
By removing this default value, it becomes necessary for board without HSE, to declare this property.

Reply:
This property is required by binding only if node is enabled (it is disabled by default).